services/user-svc/internal/organization/organization.go

```diff
@@ -794,3 +788,84 @@ func AddUserToOrganization(ctx context.Context, userId uuid.UUID, organizationId
 
 	return nil
 }
+
+func (s ServiceServer) CreatePersonalOrganization(ctx context.Context, _ *pb.CreatePersonalOrganizationRequest) (*pb.CreatePersonalOrganizationResponse, error) {
+	kc, err := hwkc.NewClientFromEnv(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	userID, err := common.GetUserID(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	userClaims, err := common.GetAuthClaims(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	organisations, err := kc.GetOrganizationsForUserById(userID)
+	if err != nil {
+		return nil, err
+	}
+
+	personalOrganizations := hwutil.Filter(organisations, func(organization *hwkc.Organization) bool {
+		return organization.Attributes.IsPersonal
+	})
+
+	if len(personalOrganizations) > 0 {
+		organizationID, err := uuid.Parse(*personalOrganizations[0].ID)
+		if err != nil {
+			return nil, fmt.Errorf("cannot parse id from keycloak for organization: %w", err)
+		}
+
+		return &pb.CreatePersonalOrganizationResponse{
+			Id: organizationID.String(),
+		}, nil
+	}
```

Comment:
You are returning a response here, but no `PersonalOrganization` has been created, right? Can you add a comment on why we do that?